__tests__/apiTopics.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+describe("Get / api/topics", () => {
+  it("returns with array of objects in topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(response.body).toBeInstanceOf(Array);
```

Comment:
Whenever we use a forEach loop, it's important we also check the length of the array - as this test could otherwise pass if it were being passed an empty array (as the forEach loop wouldn't happen), please add in a length check for the array